src/main/java/Eddie/Ui.java

```diff
@@ -0,0 +1,73 @@
+package Eddie;
+
+/**
+ * Class which handles the user interface.
+ */
+public class Ui {
+    static String welcome = "Hello! I'm Eddie\n" +
+            "What can I do for you?";
+
+    /**
+     * Prints the welcome message for when Eddie starts.
+     */
+    public static String welcome() {
+        return welcome;
+    }
+
+    /**
+     * Prints the message for when Eddie exits.
+     */
+    public static String exit() {
+        String goodbye = "Bye. Hope to see you again soon!";
+        return goodbye;
```

Comment:
You can just return the string you dont need to initialise a String before returning